LayoutTests/editing/input/mac/writing-suggestions-textarea-multiple-lines.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../../resources/ui-helper.js"></script>
+<style>
+textarea {
+    font-size: 16px;
+    width: 300px;
+    height: 300px;
+    caret-color: transparent;
+}
+</style>
+</head>
+<body>
+<textarea id="textarea">good morning.
+</textarea>
+<script>
+addEventListener("load", async () => {
+    const editor = document.getElementById('textarea');
+
+    const shadowRoot = internals.shadowRoot(editor);
+
+    shadowRoot.childNodes[0].appendChild(document.createTextNode("My na"));
```

Comment:
Is it necessary to reach into the shadow root, or can we type this out using `execCommand(…)`? If it works, I think that would more accurately reflect the user-facing steps in this bug.

Reply:
I initially tried using `execCommand`, but couldn't get the resulting shadow DOM to look quite like how it does in actuality, so I did this instead.